src/EditorFeatures/Core.Wpf/InlineHints/InlineHintsTagger.cs

```diff
@@ -5,11 +5,13 @@
 using System;
 using System.Collections.Generic;
 using System.Composition;
+using System.Linq;
```

Comment:
I don't think these new usings are actually needed at this point?

Reply:
Interestingly they weren't greyed out, must be a bug.